5_databases/week1/command.sql

```diff
@@ -0,0 +1,72 @@
+-- 1. Find out how many tasks are in the task table
+
+
+SELECT COUNT(*) 
+FROM task;
+
+-- 2. Find out how many tasks in the task table do not have a valid due date
+
+SELECT id, due_date
+FROM task
+WHERE due_date IS NULL 
+   OR STR_TO_DATE(due_date, '%Y-%m-%d %H:%i:%s') IS NULL;
+
+
+
+-- 3. Find all the tasks that are marked as done
```

Comment:
Awesome interpretation. I would probably just think more simple and check the status table first and check status_id=3 directly, but this works as well